Makefile

```diff
@@ -38,5 +22,13 @@ install:
 	go install ./cmd/ebcli
 	go install ./cmd/ebrelayer
 
-test:
-	gotestsum
\ No newline at end of file
+lint:
+	@echo "--> Running linter"
+	golangci-lint run
+	@find . -name '*.go' -type f -not -path "./vendor*" -not -path "*.git*" | xargs gofmt -d -s
+	go mod verify
+
+test_app:
```

Comment:
to keep inline with other repos in this org, could we name this test, also no snake case, but kebab-case naming convention.

Reply:
^^